README.md

```diff
@@ -1,4 +1,49 @@
-challenge_2.8
+Todo Challenge by Joseph Cumines
 =============
 
-A Symfony project created on June 8, 2016, 12:56 am.
+This project is currently live at [todo.joeycumines.me](http://todo.joeycumines.me).
+
+## Technologies Used
+### Symfony 2.8
+As specified by the outline I used Symfony 2.8. I had never used it before so there
+was quite a learning curve, but I was pleasantly surprised by how useful and well
+designed it was.
+#### FOSUserBundle
+As specified by the outline I implemented FOSUserBundle. I used MongoDB for
+persistence, and only customized the basic style for the pages provided, all
+the forms etc are default, but I linked the registration page etc.
+#### DoctrineMongoDBBundle
+I used the doctrine mongodb bundle directly to access and manipulate the Todo items.
+### MongoDB
+I was familiar with Mongo so it was an easy choice.
+### PHPUnit
+I wrote unit testing on the API using PHPUnit. I found it easy to learn and use.
+### React and ReactDOM
```

Comment:
I have been using swagger yaml specs to generate paths then implementing them in node.js, PHP 5.6 has been where I have been doing most of my web development, they teach vanilla web development (ES5, CSS and PHP) at QUT, unless you go all the way into app dev, which I couldnt because I was a IS major.